### PR TITLE
Onboarding component

### PR DESCRIPTION
**Pull request by clarkmcadoo**
Screenshot 2024-04-11 at 5 22 18 PM
Screenshot 2024-04-11 at 5 22 36 PM

### CODE REVIEW

**Comment by clarkmcadoo** [comment]
new deploy - https://app.netlify.com/sites/docs-website-netlify/deploys/66194a07a177672b8208243f

File: src/i18n/translations/en/translation.json [hunk 1]
```diff
@@ -53,7 +53,31 @@
...
+    },
+    "onBoarding": [
+      {
+        "button": "Sign up",
+        "buttonHref": "https://newrelic.com/signup",
+        "text": "Sign up for a New Relic account to unlock powerful monitoring and observability tools.",
+        "title": "Create an account"
+      },
+      {
+        "readDocsHref": "https://docs.newrelic.com",
```
Comment (sunnyzanchi) [inline src/i18n/translations/en/translation.json:65]: this is just a placeholder i figure?

**Review by sunnyzanchi — APPROVED** [review]

**Comment by tabathadelane** [comment]
There is a little bit of responsive weirdness around 650pixels Screenshot 2024-04-12 at 3 49 42 PM

File: src/i18n/translations/en/translation.json (hunk 1 above)
Comment (clarkmcadoo) [reply src/i18n/translations/en/translation.json:58]: it is

**Review by sunnyzanchi — APPROVED** [review]

**Pull request merged**